samza-core/src/main/scala/org/apache/samza/storage/ContainerStorageManager.java

```diff
@@ -714,15 +729,15 @@ private void startSideInputs() {
     LOG.info("SideInput Restore started");
 
     // initialize the sideInputStorageManagers
-    getSideInputStorageManagers().forEach(sideInputStorageManager -> sideInputStorageManager.init());
+    getSideInputHandlers().forEach(handler -> handler.init());
```

Comment:
minor: prefer to use method reference. 
if you choose to, please do in the rest of places as well.